about.html

```diff
@@ -1 +1,135 @@
-<!-- Your markup here! -->
\ No newline at end of file
+	<!DOCTYPE html>
+	<html>
+
+		<head>
+		    <!-- Meta Stuff -->
+		  <meta charset="utf-8">
+
+		  <!-- Bootstap CSS -->
+		  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+
+		  <!-- Our CSS -->
+		  <link href="styles.css" rel="stylesheet">
+
+		</head>
+
+		<body>
+			<nav class="navbar navbar-default">
+				<div class="container-fluid padding-20">
+			    	<div class="navbar-header">
+			    		<a class="navbar-brand" href="#">
+			        		<img class = 'flag' alt="Brand" src="assets/danish_flag.gif" height="25" width="25">
+			      		</a>
+			     	</div>
+					<span class='logo-text'>Danebook</span>
+		    		<input type='text' class='searchbar' placeholder="Search for Users">
+		    		<span class='name'>Harry</span>
+		    	</div>
+			</nav>
+
+			<div class="container">
+				<div class="row">
+					<div class="col-md-12">
+						<img class="full-width cover-photo" src="assets/hogwarts_small.jpg">
+					</div>
+				</div>
+				<div class="row outlined">
+					<div class="col-md-9 col-md-offset-3">
+						<span>
+							<button class="btn btn-lg fb-nav-buttons">Timeline</button>
+							<button class="btn btn-lg fb-nav-buttons fb-nav-buttons-selected">About</button>
+							<button class="btn btn-lg fb-nav-buttons">Photos</button>
+							<button class="btn btn-lg fb-nav-buttons">Friends</button>
+							<a class="edit-profile-anchor">Edit Profile</a>
+						</span>
+					</div>
+				</div>
+				<div class="row photo-container">
+					<div class="col-md-12">
+						<img class="profile-photo" src='assets/harry_potter_small.jpg'>
+						<h3 class="profile-username">Harry Potter</h3>
+					</div>
+				</div>
+			</div>
+
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-md-12">
+						<p class='about-bar'>About
+						<button class="btn btn-primary edit-profile-button">Edit Your Profile</button>
+						</p>
+					</div>
+				</div>
+			</div>
+
+			<div class="container-fluid outlined information-panel">
+				<div class="row">
+					<div class="col-md-5">
+						<h2 class='basic-info-h2'>Basic Information</h2>
+						<br>
+						<div class="container-fluid">
+							<div class="row">
+								<span><p><b class='data-label'>Birthday:</b></p>
+								<p class='data-value'>July 31st, 1980</p></span>
+							</div>
+							<div class="row">
+								<span><p><b class='data-label'>College:</b></p>
+								<p class='data-value'>Hogwarts College</p></span>
+							</div>
+							<div class="row">
+								<span><p><b class='data-label'>Hometown:</b></p>
+								<p class='data-value'>Godrick's Hollow, England</p></span>
```

Comment:
I like the use of "data-label" and "data-value" classes, very semantic and easily understood with my eyes reading it now.